pkg/oc/admin/router/router.go

```diff
@@ -447,7 +447,7 @@ func generateProbeConfigForRouter(cfg *RouterConfig, ports []kapi.ContainerPort)
 		}
 
 		probe.Handler.HTTPGet = &kapi.HTTPGetAction{
-			Path: "/healthz",
+			Path: path,
 			Port: intstr.IntOrString{
 				Type:   intstr.Int,
 				IntVal: int32(healthzPort),
```

Comment:
Even though a path is being passed (which is either livez or healthz), the port is always healthz?

Reply:
Yes, both /livez and /healthz paths are using StatsPort
https://github.com/JacobTanenbaum/origin/blob/1f93335b3f25944c202f7f05e3e52201d07ff972/pkg/cmd/infra/router/template.go#L206 and
https://github.com/JacobTanenbaum/origin/blob/1f93335b3f25944c202f7f05e3e52201d07ff972/pkg/oc/admin/router/router.go#L446